docs/release-notes/8.7.asciidoc

```diff
@@ -10,6 +10,14 @@
 ==== Known issues
 * After alerts are generated for the first time, you may have to refresh your browser before your alert data appears on pages that use data views (for example, Timeline). Navigating between pages will not work (https://github.com/elastic/security-docs/issues/3046[#3046]).
 
+
+*{agent} upgrade from 8.6.2 to 8.7.2 can fail:*
+When upgrading {agents} running the Osquery Manager integration from 8.6.2 to 8.7.0, they can get stuck in the `Updating` state (https://github.com/elastic/elastic-agent/issues/2433[#2433]). To prevent this, delete the Osquery Manager integration from your {agents} before upgrading them to 8.7.0, then re-add it after you finish upgrading.
+
+If you already tried upgrading, these steps will still resolve the issue. However, because your {agents} are stuck in the `Updating` state, you will need to upgrade them via API instead of via Kibana. First, delete the Osquery integration using Kibana, then use the https://petstore.swagger.io/?url=https://raw.githubusercontent.com/elastic/kibana/8.7/x-pack/plugins/fleet/common/openapi/bundled.json#/default/upgrade-agent[Agent upgrade API] to tell the agents to upgrade. After the upgrade completes, add the Osquery Manager integration again.
```

Comment:
```suggestion
+
If you already upgraded your {agents}, and they're are stuck in the `Updating` state, you will need to upgrade them via API instead of {fleet}. First, remove the Osquery integration from the {agents}, then use the https://petstore.swagger.io/?url=https://raw.githubusercontent.com/elastic/kibana/8.7/x-pack/plugins/fleet/common/openapi/bundled.json#/default/upgrade-agent[Agent upgrade API] to tell the {agents} to upgrade. After the upgrade completes, add the Osquery Manager integration to the {agents} again.
+
```